fastlane/lib/fastlane/helper/s3_client_helper.rb

```diff
@@ -46,7 +46,7 @@ def delete_file(bucket_name, file_name)
       end
 
       def find_bucket!(bucket_name)
-        bucket = Aws::S3::Bucket.new(bucket_name, client: client)
+        bucket = Aws::S3::Bucket.new(bucket_name, client: nil)
```

Comment:
Was this an intended part fo this change? 🤔

Reply:
@joshdholtz craps. wrong copy and past.

Reply:
@bolom Do you have a changes that you are still looking to push to fix this? 😊

Reply:
I will do it today. @joshdholtz

Reply:
Thank you!! No rush on my part but just wanted to make sure we could get this merged for you at some point 😁